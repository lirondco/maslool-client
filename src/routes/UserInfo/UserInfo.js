import React, { Component } from "react";
import EditUserForm from "../../components/EditUserForm/EditUserForm";
import "./UserInfo.css";

export default class UserInfo extends Component {
  render() {
    return (
      <section className="UserInfo">
        <h2>User Profile</h2>
        <hr />
        <EditUserForm />
      </section>
    );
  }
}
