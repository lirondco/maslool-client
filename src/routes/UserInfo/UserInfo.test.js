import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import UserInfo from './UserInfo'

describe(`UserInfo route component`, () => {
  it('renders the complete page', () => {
    const wrapper = shallow(<UserInfo />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})