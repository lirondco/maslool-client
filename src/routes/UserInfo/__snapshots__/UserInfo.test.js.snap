// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo route component renders the complete page 1`] = `
<section
  className="UserInfo"
>
  <h2>
    User Profile
  </h2>
  <hr />
  <EditUserForm />
</section>
`;
